board/batocera/rockchip/rk3326/dts/rk3326-rg351p.dts

```diff
@@ -23,6 +20,186 @@
                 bootargs = "";
 	};
 
+	rumble {
+		compatible = "pwm-vibrator";
+		pwms = <&pwm0 0 200000000 0>;
+		pwm-names = "enable";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&pwm0_pin>;
+	};
+
+	joypad {
```

Comment:
Doesn't RG351P have built-in gamepad on USB bus instead of having it on bare GPIOs + SARADC input?